operators/deploy/crds/crownlabs.polito.it_workspaces.yaml

```diff
@@ -52,6 +52,13 @@ spec:
               prettyName:
                 description: The human-readable name of the Workspace.
                 type: string
+              autoEnroll:
+                description: AutoEnroll capability definition. If omitted, no autoenroll features will be added.
+                enum:
+                - none
```

Comment:
Isn't it possible to have an empty string for disabled feature?

Reply:
replaced `none` with `""` (empty string)